### ISSUE TITLE
transparent page.bgcolor rendering in the embedded browser is abnormal

### ISSUE THREAD

**Issue by doidio** [flet-dev/flet#2727]
**Description**

I use the Web Browser plugin to build a web GUI in the Unreal Engine game. I need to set the page.bgcolor to transparent to make the GUI page float on the 3D scene. However, when I updated the flet-fastapi from 0.19.0 to 0.20.2, the rendering of the webpage GUI became abnormal, and it seemed to be repeatedly re-rendering wrong things. I think the main problem is that the Chrome browser kernel version of Unreal Engine Web Browser plugin is older, but I can't expect Epic to quickly improve its kernel version, so can the flet keep as good adaptation as 0.19.0 version to the old browser kernel version?

My client_user_agent is `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Game/++UE5+Release-5.3-CL-29314046 UnrealEngine/5.3.2-29314046+++UE5+Release-5.3 Chrome/90.0.4430.212 Safari/537.36`

**Code example to reproduce the issue:**

```python
page.bgcolor = ft.colors.TRANSPARENT
# then use an old kernel Chrome/90.0.4430.212 open it in webpage
```

**Describe the results you received:**
abnormal webpage rendering

**Describe the results you expected:**
fully transparent background page as flet-0.19.0 already done

**Additional information you deem important (e.g. issue happens only occasionally):**

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.20.2
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page: 
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: c:\users\yipin\anaconda3\envs\py310\lib\site-packages
Requires: cookiecutter, flet-runtime, packaging, qrcode, watchdog, websocket-client, websockets
Required-by: flet-fastapi
```

**Give your `requirements.txt` file (don't `pip freeze`, instead give direct packages):**

```
(The requirements)
```

**Operating system**:
Windows

**Additional environment details:**

**Comment by doidio** [comment]
I test flet-0.20.0 and it works as good as 0.19.0 but 0.20.1 not.

**Comment by ndonkoHenri** [comment]
Can you please try the [latest pre-release](https://flet.dev/blog/introducing-flet-1-0-alpha/#trying-flet-10-alpha) and let know if the issue still happens?
